src/lib/mina_base/zkapp_command.ml

```diff
@@ -1655,7 +1666,8 @@ struct
   let forget (t : t) : T.t = t.zkapp_command
 
   let to_valid (t : T.t) ~ledger ~get ~location_of_account : t Or_error.t =
-    Verifiable.create t ~ledger ~get ~location_of_account
+    Verifiable.create t
+      ~find_vk:(Verifiable.find_vk_via_ledger ~ledger ~get ~location_of_account)
```

Comment:
Does it make sense to pass this in as an argument? Looking in the ledger is not always the right thing to do (although it's possible this is only used in tests..)